### DISCUSSION TITLE
Comment activer ox-koma-letter ?

### DISCUSSION THREAD

**Discussion by raylemon** [abougouffa/minemacs#107] [Q&A]
J’aimerais écrire mes lettres avec org, en suivant ce tuto: [https://one-octet.dev/posts/org-mode-lettre.html](https://one-octet.dev/posts/org-mode-lettre.html). Il est indiqué d’importer ox-koma-letter. Comment puis-je faire ? 

Merci !

**Comment by raylemon** [comment]
J’ai trouvé, merci.

**Comment by abougouffa** [comment]
Oops, je n'ai pas vu votre question!

Je ne connaissais pas ce module, je viens de le découvrir, merci pour l'info.

Je viens de l'activer par défaut ebfdd71d4239113aa336e10e654cdd648a0de7fb!
